app/src/main/java/org/dhis2/usescases/login/LoginPresenter.kt

```diff
@@ -469,6 +469,13 @@ class LoginPresenter(
             ?.set(activate.toString())
     }
 
+    private fun deletePin() {
```

Comment:
maybe, at some pint we can think of creating a userRepository and abstract d2 for new developments like this.